test/Interop/CxxToSwiftToCxx/span/span-execution.cpp

```diff
@@ -0,0 +1,133 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+
+// RUN: %target-swift-frontend -typecheck %t/use-span.swift -typecheck -module-name UseSpan -emit-clang-header-path %t/UseSpan.h -I %t -enable-experimental-cxx-interop -Xcc -std=c++20 -clang-header-expose-decls=all-public -disable-availability-checking
```

Comment:
Do we rely on `-disable-availability-checking` here? Does the test fail with this flag removed? (here and on the next line)

Reply:
You're correct. It does not